packages/starknet-snap/src/utils/snapUtils.ts

```diff
@@ -34,7 +35,6 @@ import {
   MAXIMUM_TOKEN_SYMBOL_LENGTH,
   PRELOADED_NETWORKS,
   PRELOADED_TOKENS,
-  STARKNET_MAINNET_NETWORK,
   STARKNET_SEPOLIA_TESTNET_NETWORK,
```

Comment:
Shall we also remove this:  
 
```suggestion
```

And then use `constants.StarknetChainId.SN_SEPOLIA` instead of `STARKNET_SEPOLIA_TESTNET_NETWORK.chainId` for consistency ?

Reply:
not should what you referring to, but i believe it is not related to this PR? 
but indeed, i agree if we know the network, we dont need to use our own constant to get the ID